src/index.css

```
@@ -2,6 +2,18 @@
 @tailwind components;
 @tailwind utilities;
 
+@font-face {
+  font-family: 'Minecraftia';
```

Stylelint (17.14.0)

[error] Expected no quotes around "Minecraftia" (font-family-name-quotes)

(font-family-name-quotes)